src/device_info.sh

```diff
@@ -156,18 +188,20 @@ function get_disk()
   local cmd
   local fs
 
-  flag=${flag:-'SILENT'}
```

Comment:
Is there a reason for deleting this line?

Reply:
We should get `flag` as a parameter at this function level.

Reply:
Sure, it makes sense 😄